_includes/v19.1/app/basic-sample.clj

```diff
@@ -3,8 +3,14 @@
             [test.util :as util]))
 
 ;; Define the connection parameters to the cluster.
-(def db-spec {:subprotocol "postgresql"
-              :subname "//localhost:26257/bank"
+(def db-spec {:dbtype "postgresql"
+              :dbname "bank"
+              :host "localhost"
+              :port "26257"
+              :ssl true
+              :sslmode "require"
+              :sslcert "certs/client.maxroach.crt"
+              :sslkey "certs/client.maxroach.key.pk8"
               :user "maxroach"
               :password ""})
```

Comment:
This `:password` field is not needed.  Perhaps we should remove it just to make it a little clearer that we are not using pw-based auth.

Reply:
You're right... If you try to use the wrong key type (not the .pk8), `lein` with give you an error that you need to specify a password. I don't think this is worth mentioning though. I'm sure you ran into the same type of thing when you created the secure Java example.